app/views/custom_styles/_primer_color_mapping.erb

```diff
@@ -17,43 +17,57 @@
     --button-primary-bgColor-hover: var(--button--primary-background-hover-color) !important;
     --button-primary-bgColor-disabled: var(--button--primary-background-disabled-color) !important;
   }
-  [data-light-theme=light_high_contrast]{
+
+  /* Generic color mapping for content variables */
+  [data-light-theme=light_high_contrast],
+  [data-dark-theme=dark] {
     --avatar-border-color: var(--avatar-borderColor);
     --list-item-hover--border-color: var(--control-transparent-borderColor-hover);
-    --list-item-hover--color: var(--fgColor-default);
+    --list-item-hover--color: var(--color-action-list-item-default-hover-bg);
     --inplace-edit--border-color: var(--borderColor-default);
     --ck-color-toolbar-border: var(--borderColor-default);
     --ck-color-base-border: var(--borderColor-default);
+    --ck-color-button-default-hover-background: var(--color-action-list-item-default-hover-bg);
+    --ck-color-toolbar-background: var(--body-background);
+    --ck-color-text: var(--body-font-color);
     --table-border-color: var(--borderColor-default);
     --toolbar-item--bg-color: var(--button-default-bgColor-rest);
     --toolbar-item--border-color: var(--borderColor-default);
-    --header-border-bottom-width: 1px;
-    --header-bg-color: var(--page-header-bgColor);
-    --header-item-font-color: var(--fgColor-default);
-    --header-item-bg-hover-color: var(--control-transparent-bgColor-hover);
-    --header-item-font-hover-color: var(--fgColor-default);
-    --header-border-bottom-color: var(--borderColor-muted) !important;
-    --main-menu-bg-color: var(--page-header-bgColor);
-    --main-menu-hover-font-color: var(--fgColor-default);
-    --main-menu-bg-selected-background: var(--control-transparent-bgColor-hover);
-    --main-menu-bg-hover-background: var(--control-transparent-bgColor-hover);
-    --main-menu-font-color: var(--fgColor-default);
-    --main-menu-selected-font-color: var(--fgColor-default);
-    --main-menu-border-color: var(--borderColor-muted);
-    --main-menu-resizer-color: var(--borderColor-muted);
-    --main-menu-bg-selected-border: var(--borderColor-muted);
-    --main-menu-hover-border-color: var(--fgColor-default) !important;
-    --main-menu-fieldset-header-color: var(--fgColor-muted) !important;
-    --main-menu-border-width: 1px;
-    --button-default-borderColor-rest: var(--borderColor-default);
+    --button--border-color: var(--color-btn-border);
```

Comment:
`--color-btn-border` does also not exist any more. It is now named `--button-default-borderColor-rest`